### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by vanschelven** [timonweb/django-tailwind#208]
I hate to be "that guy", but after 10 months of no activity it's a fair question I suppose. 

Hoping the answer's "no", and if not, perhaps this issue can be a place where next steps can be discussed.

**Comment by vanschelven** [comment]
It's 3½ months, and 17 thumbs-up, later now. "Going once, going twice"...

Also: any volunteers to take over the project?

**Comment by ttsutiya** [comment]
I just found this [post](https://forum.djangoproject.com/t/tailwind-css-and-nodejs-build-tools-in-django-docker-container/2788/6) from the author of the project in the django project.

So I would say its dead.

**Comment by vanschelven** [comment]
I've set up a new home (or maybe more like a hotel or campsite?) [here](https://github.com/bugsink/django-tailwind).

I've merged all PRs that were easily mergeable, happy to take new PRs over there.

Next up is docs & pypi, let's take this step by step though.

**Comment by timonweb** [comment]
Hi there, the project isn't dead, I just haven't had time recently to work on it. If anyone wants to jump in and help with issues and PRs, let me know.

**Comment by vanschelven** [comment]
[Hooray!](https://github.com/timonweb/django-tailwind/pull/214)

**Comment by hra42** [comment]
2 months later we got no step further on keeping this project alive. I think it would be best to have a fork of this.

**Comment by vanschelven** [comment]
> fork

I did set up [a "temporary" fork](https://github.com/bugsink/django-tailwind) last time... it could serve as a starting point.

I kinda paused that whole effort on Feb 3, but even the [single buttonclick](https://github.com/timonweb/django-tailwind/pull/214) was too much I'm afraid...

**Comment by hra42** [comment]
yeah prob best to continue with this fork and move forward on pypi.

Or if @timonweb would consider adding more maintainers to this project so we can continue working on it?

Doesn't make much sense currently to pick up a project that has no active maintainer.

**Comment by vanschelven** [comment]
As of yesterday there seems to be some activity again. I think some of the open questions still make some sense though (e.g. adding more maintainers)
